index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<div class="container">
+		<article id="article1" class="article">
+			<header id="header" class="cardTitle">
+				<h1>Sabretooth Sprocket</h1>
+			</header><!-- /header -->
+			<section class="section1">
+				<img src="images/sabretooth-blk.jpg" alt="Sabretooth black sprocket" class="productImage">
+				<p class="description">Bacon ipsum dolor amet porchetta tenderloin turducken cow pork chop meatball brisket picanha ball tip bacon flank filet mignon. Tail kielbasa spare ribs, sausage chicken turducken drumstick shankle swine sirloin ribeye t-bone. Turkey tail salami leberkas pig. Filet mignon ground round shankle leberkas sausage.</p>
+				<h3 class="availability">Available</h3>
+			</section>
+			<section class="section2">
+				<header id="" class="specifications">
+					<h3>Specifications</h3>
+				</header><!-- /header -->
+				<p class="size">Size: 8" dia.</p>
+				<p class="weight">Weight: 18 oz.</p>	
+				<footer>
+					<p>Specifications valid until 7/23/16</p>
+				</footer>	
+			</section>
+			<section class="section3">
+				<header id="" class="pricingTitle">
+					<h3>Pricing</h3>
+				</header><!-- /header -->
+				<p class="priceQty">1: $4.99</p>
+				<p class="priceQty">2-50: $4.49</p>
+				<p class="priceQty">51+: $3.99</p>
+			</section>
+		</article>	
+
+		<article id="article1" class="article">
```

Comment:
Duplicate `id` values will cause confusion. Additionally, don't name `id` or `class` values by their order in the HTML. Make them appropriate for the content that the element will contain.